src/Microsoft.DotNet.CodeFormatting.Tests/FormatOptionsParsingTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+
+using CodeFormatter;
```

Comment:
would be glad to make a scrub for this later but let's loop in jared/Priya. this code is a rename/updating of existing tests, so i'll leave the NS alone for now